tools/tiara/tiara.xml

```diff
@@ -78,38 +77,47 @@
         </section>
     </inputs>
     <outputs>
-        <collection name="output" type="list" label="${tool.name} on ${on_string}: classified sequences in txt and Fasta Output">
-            <discover_datasets pattern="__name_and_ext__"  ext="fasta,txt" directory="results" />
+        <collection name="output_fasta" type="list" label="${tool.name} on ${on_string}: Fasta Output">
+            <discover_datasets pattern="__name_and_ext__"  ext="fasta" directory="results" />
+        </collection>
+        <collection name="output_txt" type="list" label="${tool.name} on ${on_string}: Classified sequences in txt">
+            <discover_datasets pattern="__name_and_ext__"  ext="txt" directory="results" />
```

Comment:
`__name_and_ext__` will collect all files and use for the `name` the filename and for the extension the `.txt` or `.fasta`. It will not filter the folder I think. You can test this in your tests.

I think what you need to do it to move all `fasta` files into a separate dir and just collect them, this is IMHO the easiest way as you already move the fasta file with your find step.